mix.exs

```diff
@@ -22,9 +22,9 @@ defmodule HTTPoison.Mixfile do
 
   defp deps do
     [
-      {:hackney, "~> 1.6.0"},
+      {:hackney, "~> 1.6.0 and >= 1.6.3"},
```

Comment:
But actually we always(or almost) had `~> MAJOR.MINOR.BUGFIX` set. Maybe we should keep it as it is? so just `~> 1.6.3`